### DISCUSSION TITLE
Next Ship To Master?

### DISCUSSION THREAD

**Discussion by jakepenn** [fnc12/sqlite_orm#1294] [Q&A]
I see that subqueries / common tables expressions are available on dev but not main -- I'd like to use them though, is there any plans to ship a version bump soon?

**Comment by fnc12** [comment]
yes we are making the last updates and then shipping v1.9

**Reply by trueqbit** [reply]
@jakemumu What is preventing you from using the development branch (or a specific commit of it)?

**Reply by jakepenn** [reply]
apologies I saw this was duplicated -- but generally I don't touch dev branches much?

**Reply by trueqbit** [reply]
I know that a tagged version has a high value, and I usually like official releases as well.
However, I can assure you that the development branch of sqlite_orm is currently in pretty good shape - all contributions are reviewed and usually verified by unit tests run by three major compilers in different C++ language modes.
